examples/SpringBootHikariExample/README.md

```diff
@@ -0,0 +1,145 @@
+# Tutorial: Getting started with Spring Boot, Hikari, and the AWS Advanced JDBC Wrapper Driver
+
+In this tutorial, you will set up a Spring Boot application using Hikari and the AWS Advanced JDBC Wrapper Driver.
+
+> Note: this tutorial was written using the following technologies:
+>    - Spring Boot 3.1.0
+>    - AWS Advanced JDBC Wrapper 2.1.2 
+>    - Postgresql 42.5.4
+>    - Kotlin
+>    - Java 8
```

Comment:
https://spring.io/blog/2022/05/24/preparing-for-spring-boot-3-0

Spring 3 requires Java 17

Reply:
Done!